### PR TITLE
strategy providers list view table row changes

### PR DESCRIPTION
**Pull request by digvijay-gurav**

### CODE REVIEW

File: LESS/navigation.less [hunk 1]
```diff
@@ -800,4 +800,33 @@ input.startegy-providers-input-box {
   font-size: 24px;
   font-weight: 600;
   margin: 0;
+}
+.ibox-content .table .return-percentage {
```
Comment (neerajdembla212) [inline LESS/navigation.less:804]: these css should be either in `custom.less` or create another file for strategic-provider page and import in style.less

File: css/style.css [hunk 2]
```diff
@@ -1531,6 +1531,35 @@ input.startegy-providers-input-box {
...
+.table .pt-pb-20 {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+.table .btn-width-90 {
+  width: 90%;
+}
+.table .border-box {
+  width: 50%;
+  border: 1px solid #E7EDF4;
```
Comment (neerajdembla212) [inline css/style.css:1555]: is this color picked from sigma ? if yes, create a variable @table-border-color in `variables.less` and use it here.

**Review by neerajdembla212 — CHANGES_REQUESTED** [review]

**Review by digvijay-gurav — COMMENTED** [review]
Made the suggested changes

File: LESS/navigation.less (hunk 1 above)
Comment (digvijay-gurav) [reply LESS/navigation.less]: Made the suggested changes

File: css/style.css (hunk 2 above)
Comment (digvijay-gurav) [reply css/style.css:1532]: Made the suggested changes

**Review by neerajdembla212 — APPROVED** [review]

**Pull request merged**